fastlane/Fastfile

```diff
@@ -650,15 +708,24 @@ REPOSITORY_NAME="WordPress-Android"
   lane :build_bundle do | options |
     # Create the file names
     version=options[:version]
-    name="wpandroid-#{version["name"]}.aab"
+    app = options[:app]
+
+    if version.nil?
+      UI.message("Version specified for #{app} bundle is nil. Skipping ahead")
+      next
+    end
+
+    prefix = app.downcase == "jetpack" ? JP_PREFIX : WP_PREFIX
+    name="#{prefix}-#{version["name"]}.aab"
     apk_name=universal_apk_name(version)
-    aab_file="org.wordpress.android-#{options[:app]}-#{options[:flavor]}-#{options[:buildType]}.aab".downcase
+
+    aab_file="org.wordpress.android-#{app}-#{options[:flavor]}-#{options[:buildType]}.aab".downcase
```

Comment:
nitpicking, can wait a future PR:

is there a reason why the aab file is not using the package name as prefix, i.e. start with `com.jetpack.android-` instead of `org.wordpress.android` for the case of Jetpack?

It doesn't really matter in practice, but just for consistency…

Reply:
This is just the generated name used by gradle when running `./gradlew bundle..` we could look into ways to align this more with flavors but not sure of any way right out the box, maybe @oguzkocer  could have an idea on this one!